project2-beat-that/script.js

```diff
@@ -1,4 +1,299 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var rollDice = function () {
+
+  var randomDecimal = Math.random() * 6;
+
+  var randomInteger = Math.floor(randomDecimal);
+
+  var diceNumber = randomInteger + 1;
+
+  return diceNumber;
+};
```

Comment:
we showed this function line by line to make it clearer what is happening, but once you understand it, i think its succinct enough that the body can be reduced to one line

`return Math.floor(Math.random() * 6) + 1;`